inspire/base/static/js/settings.js

```diff
@@ -100,6 +101,10 @@ require.config({
       deps: ["jquery"],
       exports: "$.fn.tokenInput"
     },
+    "dropdown-menu": {
+      deps: ["jquery"],
+      exports: "$.fn.menuAim"
```

Comment:
This looks a bit strange. `$.fn.menuAim` should be exported by `jQuery-menu-aim` and not by that file.

If you add  `jQuery-menu-aim` as a dependency as mentioned on top you will remove from this file all references to `dropdown-menu` and instead add the configuration for  `jQuery-menu-aim`